### ISSUE TITLE
maillage : choix par territoire

### ISSUE THREAD

**Issue by lpojgc** [lpoaura/BirdAtlasOfFrance#291]
pour SPM, on est sur le maiallage 5km, Nouvelle Calédonie c'est 10km et Polynésie française - pas déifnit.
liste à finaliser pour les autres territoires

**Comment by lpofredc** [comment]
SPM: OK, on est déjà sur 5x5

https://github.com/lpoaura/BirdAtlasOfFrance/blob/694ee2bfbb2f3063b3a5934c41700eadb52cabe1/data/core/ref_geo/grids.txt#L11

Nouvelle calédonie aussu en 10x10
https://github.com/lpoaura/BirdAtlasOfFrance/blob/694ee2bfbb2f3063b3a5934c41700eadb52cabe1/data/core/ref_geo/grids.txt#L6

Polynésie française, actuellement en 5x5

https://github.com/lpoaura/BirdAtlasOfFrance/blob/694ee2bfbb2f3063b3a5934c41700eadb52cabe1/data/core/ref_geo/grids.txt#L7

**Comment by lpojgc** [comment]
|Territoire | Maillage|
|---|---|
|Saint Barthélémy | 1 km|
|Saint Martin | 1 km|
|Clipperton | ?|
|Polynésie Fr|?|
|Walis et Futuma | 5 km|
|TAAF sub|10|
|TAAF éparses | ?|
(info de JD février)

**Comment by lpofredc** [comment]
Voici le maillage actuellement utilisé sur odf.

Territoires | Taille des mailles installées (m) | Remarques
-- | -- | --
France métropolitaine | 10000 |  
Saint-Martin | 1000 |  
Saint-Barthélémy | 1000 |  
Guadeloupe | 2000 |  
Martinique | 2000 |  
Guyane | 10000 |  
Polynésie française | 5000 | Pas adapté pour les atolls
Wallis & Futuna | 5000 | Pas adapté
Mayotte | 5000 |  
TAAF | 10000 | Pas adapté pour les petites îles
Réunion | 5000 |  
Nouvelle calédonie | 10000 |

**Comment by lpojgc** [comment]
@lpofredc peux-tu préciser tes attentes ? que @JeremyDupuy te donne un maillage (ou autre polygone) pour les 3 lignes de ton tableau, c'est ça ?

**Comment by lpofredc** [comment]
Oui, c'est bien ça. Soit choisir un maillage plus fin, soit un autre type de représentation cartographique (L'île entière par exemple ?)

**Comment by JeremyDupuy** [comment]
Pour Clipperton: une seule entité pour l’îlot

Polynésie-Française: Tahiti (5km), Moorea-Maiao (5km), Bora-Bora (5km), Tahaa, Raiatea, Huanine-Nui (5km)
Pour les autres de PF à l'île. le problème, c'est que je n'ai pas de couche SIG bien adapté à ces territoires. Faut que je vois avec l'Atlas Oiseaux marin OM s'il y a une couche propre.

TAAF:
Kerguelen: 10km
Crozet: à l'île
Saint-Paul: île
Amsterdam: île
Eparses: îles
Terre Adélie: 10km

**Comment by lpojgc** [comment]
@lpofredc peux-tu lister tous les zonages qu'il manque pour que @JeremyDupuy puisse les fournir ?

**Comment by lpojgc** [comment]
données manquantes : petites îles de Polynésie française
